std_msgs/Int8 flag
---
bool received